2.0/contrib/src/.bashrc

```diff
@@ -1,2 +1,3 @@
 # This will make scl collection binaries work out of box.
+unset BASH_ENV
```

Comment:
@mnagy @bparees FYI you have to do this to avoid nasty recursions (@mnagy, please update database images)